# the question being asked
std_msgs/String question

# the context paragraph
std_msgs/String context